spec/swagger.yaml

```diff
@@ -747,71 +646,94 @@ paths:
-          required: true
-        - name: include
-          description: List of related resources to include in the response
+          description: |
+            Patient id to fetch calendar event. Note that one of the following filters must be specified: `filter[patient]`, `filter[group]`, `filter[organization]`, or `filter[attendees]`.
+        - name: filter[groups]
+          in: query
+          type: string
+          description: |
+            Comma-separated list of group ids. Note that one of the following filters must be specified: `filter[patient]`, `filter[group]`, `filter[organization]`, or `filter[attendees]`.
+        - name: filter[organization]
+          in: query
+          type: string
+          description: |
+            Twine organization id. Note that one of the following filters must be specified: `filter[patient]`, `filter[group]`, `filter[organization]`, or `filter[attendees]`.
+        - name: filter[attendees]
+          in: query
+          type: string
+          description: |
+            Comma-separated list of coach or patient ids. Note that one of the following filters must be specified: `filter[patient]`, `filter[group]`, `filter[organization]`, or `filter[attendees]`.
+        - name: filter[type]
           in: query
-          required: false
           type: string
+          description: Calendar event type
           enum:
-            - actions
-            - bundles
-            - patient
-            - current_results
-      responses:
-        '200':
-          description: OK
-          schema:
-            $ref: '#/definitions/FetchPatientPlanSummariesResponse'
-        '401':
-          description: Unauthorized
-          schema:
-            $ref: '#/definitions/FetchErrorResponse'
-        '403':
-          description: Forbidden
-          schema:
-            $ref: '#/definitions/FetchErrorResponse'
-  /patient_plan_summary/{id}:
-    get:
-      tags:
-        - plan
-      summary: Get the plan summary for a patient
-      description: Get the plan summary for a patient.
-      operationId: fetchPatientPlanSummary
-      parameters:
-        - name: id
-          in: path
-          description: Plan summary identifier
-          required: true
+            - plan-check-in
+            - reminder
+            - telephone-call
+            - office-visit
+            - video-call
+        - name: filter[completed]
+          in: query
+          type: boolean
+          description: If not specified, return all calendar events. If set to `true` return only events marked as completed, if set to `false`, return only events not marked as completed yet.
+        - name: filter[start_at]
+          in: query
+          type: string
+          description: The start (inclusive) and end (exclusive) dates in YYYY-MM-DDTHH:MM:SSZ format separated by a `..` to filter events by the `start_at` date (example `filter[start_at]=2017-11-01T00:00:00Z..2017-12-01T00:00:00Z`)
```

Comment:
The dates need to be in ISO8601 format, which can include a time zone offset or Z for UTC, so saying "YYYY-MM-DDTHH:MM:SSZ" format is not quite accurate.

Also, your example will include all calendar events that start in November 2017 UTC, but not November 2017 in local time for a time zone such as America/New_York. This is misleading. Give the example using typical values for America/New_York instead: `2017-11-01T00:00:00.000-04:00..2017-12-01T00:00:00-05:00`. Useful tool: http://dencode.com/date/iso8601